docs/examples/pulsar.md

```diff
@@ -0,0 +1,114 @@
+# Pulsar
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"testing"
+	"time"
+
+	"github.com/apache/pulsar-client-go/pulsar"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+func TestPulsar(t *testing.T) {
```

Comment:
I think it's fine to have the log consumer in place to demonstrate how to do a test.

What other examples do is to create a method to setup the container (setupPulsar) which only creates the container, and then they create a TestFoo function with the real test.

I think that would be even more helpful, as we decouple the pulsar container creation from its usage in a test. Wdyt?

Reply:
Done :)